pandas/tseries/index.py

```diff
@@ -795,20 +818,20 @@ def to_series(self, keep_tz=False):
         Parameters
         ----------
         keep_tz : optional, defaults False.
-                  return the data keeping the timezone.
+          return the data keeping the timezone.
```

Comment:
Not that important, but we normally also use indentation of 4 in the docstrings for the parameter explanation